workers/unity/.gitignore

```diff
@@ -3,9 +3,11 @@
 /[Ll]ibrary/
 /[Tt]emp/
 /Logs/
+/UserSettings/
 /ProjectSettings/UnityConnectSettings.asset
 /*.csproj
 /*.sln
+/Packages/packages-lock.json
```

Comment:
I think this is intended to be commited into the repo

Reply:
Discussed offline - we won't commit the lock to test transitive dependency changes which users could run into.